runtime/bref/0.2/serverless.yaml

```diff
@@ -0,0 +1,50 @@
+# Read the documentation at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: my-symfony-app
+configValidationMode: error
+
+provider:
+    name: aws
+    # The AWS region in which to deploy (us-east-1 is the default)
+    region: us-east-1
+    # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
+    stage: dev
```

Comment:
I would named it prod as APP_ENV below is set to prod.

Reply:
In my mind, Serverless stages and Symfony's `APP_ENV` are different. I'm often running several stages (dev/staging/prod) with `APP_ENV=prod`. But as `dev` is the default in Serverless framework, maybe this line could be totally removed ?